src/components/VideoGallery.tsx

```diff
@@ -82,16 +70,15 @@ export const VideoGallery: React.FC<VideoGalleryProps> = ({
       paddingLeft={sizes(2, true)}
       paddingTop={sizes(2, true)}
       responsive={breakpoints}
-      itemWidth={MIN_VIDEO_PREVIEW_WIDTH}
-      arrowPosition={arrowPosition}
+      itemWidth={281}
```

Comment:
Let's store this magic number in some constant variable, above component declaration.